index.bs

```diff
@@ -78,4 +84,32 @@ interface StorageBucket {
 };
 </xmp>
 
+<h3 id="storage-bucket-durability">Durability</h3>
+
+A {{StorageBucket}} has a <dfn>bucket durability</dfn>. This reflects whether the user agent
+will prioritize performance or durability when completing operations on data in the bucket.
+The value is one of the following:
+
+: "{{StorageBucketDurability/strict}}"
+:: The user agent MUST consider that operations on data in the bucket are successful only after verifying that outstanding changes have been written to a persistent storage medium.
+: "{{StorageBucketDurability/relaxed}}"
+:: The user agent MUST consider that operations on data in the bucket are successful as soon as all outstanding changes have been written to the operating system, without subsequent verification.
+
+The user agent MUST ignore the [=bucket durability=] for operations that otherwise override default durability behavior.
+
+<h3 id="storage-bucket-indexeddb">Creating an Indexed Database</h3>
+
+Each {{StorageBucket}} has an {{IDBFactory}} object. {{IDBDatabase}} objects created by
+{{StorageBucket/indexedDB}} are associated with the bucket.
+
+The user agent should consider the associated {{StorageBucket/durability}} when evaluating the <a spec="IndexedDB">durability hint</a> |durability|
```

Comment:
Oh, just noticed. Should this be surrounded by `<div algorithm></div>`?

Seems like a common pattern I see when explaining method steps ex [1].

[1] https://github.com/w3c/IndexedDB/blob/main/index.bs#L1114